### ISSUE TITLE
TrollStore versions problem

### ISSUE THREAD

**Issue by k3v1nwrld** [m1337v/RootHideManagerApp#1]
The TrollStore versions of the app won't delete files/folders in /var. I only noticed /var, but there may be some other locations too. I know /var/mobile works but /var I am sure that it does not work

**Comment by m1337v** [comment]
You are right, just tested it by creating a new file "test.sh" as well as a blacklisted folder "ssh". Both were not removed.
I currently don't have the capacity to look into it in detail. Will get to it once i have more time. 
For the time being you can install Filza and manually delete the respective files

**Comment by m1337v** [comment]
It's apparently due to the files being owned by root, for which we don't have permission to delete

**Comment by m1337v** [comment]
should be fixed with the latest release
